blocks/api/categories.js

```diff
@@ -27,3 +30,105 @@ const categories = [
 export function getCategories() {
 	return categories;
 }
+
+/**
+ * Register a new block category
+ *
+ * @param {Array} category e.g {slug: 'custom', title: __('Custom Blocks')}
+ *
+ * @return {Array} Block categories
+ *
+ */
+export function registerCategory( category ) {
+	if ( ! category ) {
+		console.error(
+			'The Block category must be defined'
```

Comment:
'The `block Category` must be defined' 

Having
-  `b` lowercase for `block` seems right. 
- `C` upercase for `Category` seems right.

Maybe change that everywhere?